src/main/java/com/benko/exercise/sort/BubbleSorter.java

```diff
@@ -0,0 +1,31 @@
+package com.benko.exercise.sort;
+
+public class BubbleSorter {
+
+    public void sort(int[] numbers) {
+        for (int i = 0; i < numbers.length; i++) {
+            swapElements(numbers);
+        }
+    }
+
+    private static void swapElements(int[] numbers) {
+        for (int i = 0; i < numbers.length-1; i++) {
+            if (isGreaterThan(i, i+1, numbers)) {
+                int temp;
+                temp = numbers[i + 1];
+                numbers[i + 1] = numbers[i];
+                numbers[i] = temp;
+            }
+        }
+    }
+
+    private static boolean isGreaterThan(int x, int y, int[] numbers) {
```

Comment:
Consider renaming parameters `x` and `y` to something more descriptive. My proposition: `int current` and `int next`.

Reply:
done